codebreaker_console/helpers/statistics.rb

```diff
@@ -0,0 +1,32 @@
+class Statistics
+  DB_PATH = 'db/results.yml'.freeze
+
+  def self.load
+    YAML.load_file(DB_PATH)
+  rescue SystemCallError
+    []
+  end
+
+  def self.save(data)
+    results = self.load
+    results << data
+
+    File.write(DB_PATH, sorted(results).to_yaml)
+  end
+
+  def self.show
```

Comment:
statistics is not the class for the presentation. Use it just for data. If you need presentor for it - create